fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -4127,7 +4130,7 @@ class FileBackupAgentImpl {
 		Reference<IBackupContainer> bc = wait(backupConfig.backupContainer().getOrThrow(cx));
 
 		TraceEvent("AS_StartRestore");
-		Version ver = wait( restore(backupAgent, cx, tagName, KeyRef(bc->getURL()), ranges, true, -1, true, addPrefix, removePrefix, true, randomUid) );
+		Version ver = wait( restore(backupAgent, cx, cx, tagName, KeyRef(bc->getURL()), ranges, true, -1, true, addPrefix, removePrefix, true, randomUid) );
```

Comment:
I think you meant `... cx, cxOrig, ...`

Reply:
No, atomicRestore() is a backup/restore test that uses a single database which is both the source of a backup and the destination for a restore of that backup.  It takes a single database handle with a running backup, locks the database, completes the backup, restores the backup, then unlocks the database.